test/test_sparse_csr.py

```diff
@@ -2530,112 +2544,190 @@ def _to_from_layout(layout_a, layout_b):
 
     @skipMeta
     @all_sparse_compressed_layouts()
+    @batched_nonbatched()
+    @hybrid_nonhybrid()
     @unittest.skipIf(not TEST_SCIPY, "SciPy not found")
-    def test_dense_to_from_sparse_compressed(self, device, layout):
+    def test_dense_to_from_sparse_compressed(self, device, hybrid, batched, layout):
         """
         This test tests conversion from dense to/from CSR and CSC
         by comparing to SciPy's implementation.
 
         TODO: Eventually this is meant to be merged into test_compressed_layout_conversions_coverage
         """
-        if layout is torch.sparse_bsc:
-            # TODO: Remove this once support has been enabled
-            return
 
-        shapes = [(6, 10), (0, 10), (6, 0), (0, 0)]
+        # adjust this block as support is added
+        supports_batched_from_sparse = (torch.sparse_bsr, torch.sparse_bsc)
+        supports_batched_to_sparse = (torch.sparse_bsr, torch.sparse_bsc)
+        supports_hybrid_from_sparse = ()
+        supports_hybrid_to_sparse = ()
 
-        blocksizes = [(2, 2)]
-        batch_sizes = [(3, )]
+        if hybrid:
+            # todo: remove this once we guard methods against hybrid dims, currently segfualts
+            raise unittest.SkipTest("Hybrid sparse tensors are not supported for this layout")
```

Comment:
we're guarding in the implementations though, right? would we want to test that they throw rather than skipping the test?

Reply:
As it stands the implementations for some layouts are not properly guarded. I will be adding this as I go, but you can call `hybrid_csr.to_dense()` for example and it causes a segfault.

Reply:
yeahh I've run into this one myself while playing around. I guess I'm thinking that it would be nice to add an expect-throw test to cover the check you added in this pr IIRC, but if it'll be complicated to thread the needle in the tests, maybe best to leave it until all the guards have been added

Reply:
After this, there will be a change to add batched support to CSR/CSC and then all layouts should be guarded against calls with dense dimensions, so the test will be updated to expect throws.